docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,28 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+:::info
+💡 According to [the React docs](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
+:::
```

Comment:
Would it read better by grouping concerns? First, the main message, second the side info?

<img width="781" alt="Screenshot 2022-05-04 at 17 38 32" src="https://user-images.githubusercontent.com/3165635/166718348-212aa290-1ba7-447a-b3b0-254d0cb58696.png">

Reply:
Re: [Marija's comment here](https://github.com/mui/material-ui/pull/32157#discussion_r864975703) I'm thinking about moving this info message down into the "Basic usage" section and elaborating on it a bit.